.github/workflows/deploy.yml

```diff
@@ -58,3 +58,11 @@ jobs:
         env:
           CLOUDFLARE_API_TOKEN: ${{ secrets.CF_API_TOKEN }}
         run: NX_CLOUD_DISTRIBUTED_EXECUTION=false npm run deploy:beta
+
+      # We will deploy to dev so that cloudflare preview urls will point to the backend
+      # This will be a problem with multiple PRs, but hey we at least tried :)
+      - name: Deploy Dev
+        if: github.ref != 'refs/heads/beta' && github.ref != 'refs/heads/master'
```

Comment:
non-blocking: What are the odds we can restrict this action to only run for users who are a part of the polyratings org?

Reply:
I'm pretty sure new contributors need to be approved to be able to run actions. This is GitHub wide to stop GitHub actions spamming by bots